reference/7.1/Microsoft.PowerShell.Core/About/about_Functions_CmdletBindingAttribute.md

```diff
@@ -13,27 +13,27 @@ Describes the attribute that makes a function work like a compiled cmdlet.
 
 ## Long description
 
-The **CmdletBinding** attribute is an attribute of functions that makes them
-operate like compiled cmdlets that are written in C#, and it provides access to
-features of cmdlets.
+The `CmdletBinding` attribute is an attribute of functions that makes them
+operate like compiled cmdlets written in C#. It provides access to the features
+of cmdlets.
 
-PowerShell binds the parameters of functions that have the **CmdletBinding**
+PowerShell binds the parameters of functions that have the `CmdletBinding`
 attribute in the same way that it binds the parameters of compiled cmdlets. The
-`$PSCmdlet` automatic variable is available to functions with the CmdletBinding
-attribute, but the `$Args` variable is not available.
+`$PSCmdlet` automatic variable is available to functions with the
+`CmdletBinding` attribute, but the `$Args` variable is not available.
 
-In functions that have the **CmdletBinding** attribute, unknown parameters and
+In functions that have the `CmdletBinding` attribute, unknown parameters and
 positional arguments that have no matching positional parameters cause
 parameter binding to fail.
 
 > [!NOTE]
 > Compiled cmdlets use the required **Cmdlet** attribute, which is similar
```

Comment:
Is it weird that the Cmdlet attribute is bolded but CmdletBinding is code blocked? I know they are different but when reading it it looks inconsistent.

Reply:
Fixed